openhabian-setup.sh

```diff
@@ -110,6 +110,7 @@ if [[ -n "$UNATTENDED" ]]; then
   frontail_setup
   zram_setup
   exim_setup
+  install_tailscale "install" && setup_tailscale && add_admin_ssh_key
```

Comment:
Add a check in `install_tailscale` that will cancel the installation at that point if the user has not input the needed values in `openhabian.conf`  otherwise we are bloating the install process with unrequested packages by installing tailscale then canceling afterwards.

Reply:
ok like so?

Reply:
Sure